api/v1beta1/verticadb_types.go

```diff
@@ -36,6 +36,9 @@ import (
 const VerticaDBKind = "VerticaDB"
 const VerticaDBAPIVersion = "vertica.com/v1beta1"
 
+// Set Constant Upgrade Requeue Time
+const URTime = 120
```

Comment:
Can we use 30 seconds instead?  We will want to retry as quick as possible during an upgrade as we really need to get the system back up.